### DISCUSSION TITLE
Playlist submit fails on partial Classic customization

### DISCUSSION THREAD

**Discussion by itotallygetthat** [ppy/osu#18946] [Q&A] [answered]
[database.log](https://github.com/ppy/osu/files/9015688/database.log)
[network.log](https://github.com/ppy/osu/files/9015689/network.log)
[performance.log](https://github.com/ppy/osu/files/9015690/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9015691/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9015692/updater.log)

Logs from early Tuesday UTC on version 2022.621.0 , but the same issue appears to still exist in 2022.630.0

On the Playlist edit screen:
When hitting "Create", the client lags for a few seconds then crashes.    The playlist submits corrupted (name, length, time, duration are correct, but no maps are returned).

This error only seems to occur if you have the Custom mod enabled AND some of the options are toggled off.   

I can't submit any more playlists for now due to max 5 open playlists per account to test more combinations of Classic customization toggles.  Here's what I found so far:
"No slider head movement" = True, the other 3 options = False. => crash
All 4 options = True => no crash

I successfully submitted a playlist with partial Classic on 2022-06-20, so something seemed to change since version 2022.615.  (Or maybe something on the backend doesn't like the partial customization?)

**Comment by peppy** [comment, marked as answer]
Tracking via #18950

**Comment by itotallygetthat** [comment]
issue resolved
